configs/testdata/valkyrie_config.test.yml

```diff
@@ -42,14 +43,14 @@ providers:
     url: "https://caleta-url"
     base_path: "/caleta" # base path to differentiate exposed endpoints between providers
     auth:
-      verification_key: |
-        -----BEGIN PUBLIC KEY-----
-        caleta-verification-key
-        -----END PUBLIC KEY-----
-      signing_key: |
-        -----BEGIN RSA PRIVATE KEY-----
-        caleta-signing-key
-        -----END RSA PRIVATE KEY-----
+      #verification_key: |
+      #  -----BEGIN PUBLIC KEY-----
+      #  caleta-verification-key
+      #  -----END PUBLIC KEY-----
+      #signing_key: |
+      #  -----BEGIN RSA PRIVATE KEY-----
+      #  caleta-signing-key
+      #  -----END RSA PRIVATE KEY-----
```

Comment:
Commented these out, since the sample values shouldn't be used for real. Trying to use them causes the rsa key reader to panic on valkyrie startup which was a bit confusing